lib/flat_search_web/plug/local_plug.ex

```diff
@@ -0,0 +1,67 @@
+defmodule FlatSearchWeb.Plugs.Locale do
+  import Plug.Conn
+
+  def init(_opts), do: nil
+
+  def call(conn, _opts) do
+    accepted_languages = extract_accept_language(conn)
+    known_locales = Gettext.known_locales(FlatSearchWeb.Gettext)
+
+    accepted_languages =
+      known_locales --
+        (known_locales -- accepted_languages)
```

Comment:
`Enum.filter(know_locales, fn locale -> locale in accepted_languages)`?

Reply:
also you can change accepted_languages to accepted_locales, to be consistent with one naming